### PR TITLE
Update to Python 3.11 to resolve some CVEs

### PR DESCRIPTION
**Pull request by gblewis1**
This will upgrade to Python 3.11 to resolve some CVEs.

### CODE REVIEW

**Comment by gblewis1** [comment]
This will require Python 3.11.5 to upgrade OpenSSL dependency. 3.12 would also work but that requires some more work in terms of dealing with an old flake8 being pulled in and being incompatible with importlib_metadata.

Python 3.11.5 was just released so it might be a few days before it's in the deadsnakes repo.

**Comment by gblewis1** [comment]
@nmgaston Please review. I'll need to run a couple tests before merging (including integration test).

**Comment by gblewis1** [comment]
Windows (UCC) integration test passed.

**Comment by gblewis1** [comment]
Integration reloaded pass.

File: inbm-lib/pyproject.toml [hunk 1]
```diff
@@ -43,4 +43,5 @@ max_annotation_complexity = 5
 [tool.flakeheaven.plugins]
 "flake8-annotations" = ["-*", "+ANN001", "+ANN2??", "+ANN301"]
 "flake8-bandit" = ["+*", "-S404", "-S603", "-S101"]
-"wemake-python-styleguide" = ["-*", "+WPS2??", "-WPS220", "-WPS222", "-WPS223", "-WPS202"]
+"wemake-python-styleguide" = ["-*", "+WPS2??", "-WPS202", "-WPS220", "-WPS223", "-WPS222", "-WPS236", "-WPS237", "-WPS238"] # WPS2?? is complexity; WPS202 is module members; 220 is deep nesting; 223 is elifs; 222 is logic; 236 is # of variables unpacked in a tuple; 237 is complex format string; 238 is # of raises in a function
```
Comment (nmgaston) [inline inbm-lib/pyproject.toml:46]: Should we create stories to try and resolve some of these style issues?  At least try anyways.

**Review by nmgaston — APPROVED** [review]

**Pull request merged**